lib/data-types.js

```diff
@@ -844,10 +844,10 @@ class GEOMETRY extends ABSTRACT {
     this.srid = options.srid;
   }
   _stringify(value, options) {
-    return `GeomFromText(${options.escape(wkx.Geometry.parseGeoJSON(value).toWkt())})`;
+    return `ST_GeomFromText(${options.escape(wkx.Geometry.parseGeoJSON(value).toWkt())})`;
```

Comment:
Can you please update [this line](https://github.com/sequelize/sequelize/blob/6d03c09175b2b91e19b68dd200c38c085beedc2e/lib/data-types.js#L795) accordingly?

Also can you quickly explain what is the difference between these two functions? Why does mysql provide two similar-looking functions like this while one of them doesn't work for some types (which is what I am inferring)? (I never used this, could be obvious to you, sorry)

Reply:
Looks like they are synonyms, and the recommendation from MySQL is to use the ST_ prefixed versions

```
ST_GeomFromText(), ST_GeometryFromText(), GeomFromText(), and GeometryFromText() are synonyms. For more information, see the description of ST_GeomFromText().

GeomFromText() and GeometryFromText() are deprecated; expect them to be removed in a future MySQL release. Use ST_GeomFromText() and ST_GeometryFromText() instead.
```

From https://dev.mysql.com/doc/refman/5.7/en/gis-wkt-functions.html

Reply:
> Can you please update [this line](https://github.com/sequelize/sequelize/blob/6d03c09175b2b91e19b68dd200c38c085beedc2e/lib/data-types.js#L795) accordingly?

Will do!

> Looks like they are synonyms, and the recommendation from MySQL is to use the ST_ prefixed versions
> 
> ```
> ST_GeomFromText(), ST_GeometryFromText(), GeomFromText(), and GeometryFromText() are synonyms. For more information, see the description of ST_GeomFromText().
> 
> GeomFromText() and GeometryFromText() are deprecated; expect them to be removed in a future MySQL release. Use ST_GeomFromText() and ST_GeometryFromText() instead.
> ```
> 
> From [dev.mysql.com/doc/refman/5.7/en/gis-wkt-functions.html](https://dev.mysql.com/doc/refman/5.7/en/gis-wkt-functions.html)

Correct, the non-`ST_` versions have been deprecated for a few years now.